plugins/coatils.py

```diff
@@ -163,7 +165,9 @@ def position(stl, stc, enl, enc):
                                                               enl, enc)
 
     # Ignore PycodestyleBear, LineLengthBear
-    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$')
+    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$',
+               re_cmd_name_help='run <Bear [[setting=value]+]>+\n'
+                                '```\n<code>+\n```')
```

Comment:
```
run <Bear setting=value....>
code here
```

This is way simpler. Someone using the bear for the first time does not need to know the settings are optional, you could even mention it later in the docs

Another option: Keep as it is but give concrete examples of all possiblities of the regex